System/usr/trimui/scripts/load_launcher.sh

```diff
@@ -0,0 +1,29 @@
+export PATH="/mnt/SDCARD/System/usr/trimui/scripts/:/mnt/SDCARD/System/bin:${PATH:+:$PATH}"
+export LD_LIBRARY_PATH="/usr/trimui/lib:${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}"
+
+EMU_DIR="$(dirname "$0")"
+GAME=$(basename "$@")
+
+# The preset file is only created after a first saved launcher.
+if [ -e "$EMU_DIR/presets.txt" ]; then
+	# Press L to start a game with a saved launcher.
+	button_state.sh L
+	[ $? -eq 10 ] && Launcher_name=$(grep -i "$GAME" "$EMU_DIR/presets.txt" | cut -d'=' -f2)
+
+	[ -z "$Launcher_name" ] && Launcher_name=$(grep -i default "$EMU_DIR/presets.txt" | cut -d'=' -f2)
```

Comment:
ToDo: Fallback to others default finding if the presets.txt only contain games launchers but no system default.